LOCAL_DEVELOPMENT.md

```
@@ -0,0 +1,580 @@
+mysql --version      # should print: mysql  Ver 8.0...
+git --version        # should print: git version 2.x
+```
+
+---
+
+## 2. Repository Overview
+
+AMRIT is split across many repositories. For local development, you typically work
+on one module at a time. Here's the dependency map:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)